client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -628,6 +628,7 @@ export default connect(
 			jetpackSSOSessions: getSSOSessions( state ),
 			isAlreadyOnSitesList: isRemoteSiteOnSitesList( state ),
 			isFetchingSites: isRequestingSites( state ),
+			isFetchingSite: isRequestingSite( state ),
```

Comment:
`isFetchingAuthorizationSite` is good, i'll switch it for that.
With #13758, we were able to drop `isFetchingSites` entirely, but I'm trying to take smaller steps to solve the speed issue, so unfortunately, we do need both selectors.  But perhaps in a future PR, we can drop the second `me/sites` query as well, and make it a little clearer.